backend/app/workers/llm.py

```
@@ -0,0 +1,236 @@
+"""
+Deterministic LLM worker stubs with audit writes.
+
+These workers write tenant-scoped records with cost=0 and never call providers.
+"""
+
+from __future__ import annotations
+
+import logging
+from datetime import date, datetime, timezone
+from typing import Any, Dict
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)